### ISSUE TITLE
Invalid loaded license is logged as DEBUG event hiding this from the user

### ISSUE THREAD

**Issue by ramonsmits** [Particular/NServiceBus#5630]
Customer has a license file that for some reason is not loading correctly. Loading/parsing errors should not be logged with DEBUG but as WARNING.

Log after enabling DEBUG:
```txt
2020-04-28 22:10:44.280 DEBUG Looking for license in the following locations:
License found in C:/temp/license (1).xml is not valid - XML is invalid because it failed the signature check
No valid license could be found. Falling back to trial license with start date '2018-01-26'.
2020-04-28 22:10:44.289 ERROR Trial license expired. To continue using the Particular Service Platform, please extend your trial or purchase a license by visiting http://go.particular.net/trial-expired
2020-04-28 22:10:44.296 WARN  To extend your trial license, visit: https://particular.net/license/nservicebus?v=7.2.4&t=0&p=windows
```

**Comment by timbussmann** [comment]
there are log statements higher than debug to indicate license related issues. Debug statements are only used to indicate all searched locations and why a certain license in a specific folder has not been used. This was done intentionally to not create an unnecessary amount of noise unless you're trying to figure out why your configured license isn't used (= debugging). From this log statement I'd probably assume that there is a specific license folder configured instead of using the default license locations.

**Comment by ramonsmits** [comment]
@timbussmann The issue is parsing/deserializing. That probing locations are logged as DEBUG is fine but the fact that after a probe a found license could not be converted or is expired should not DEBUG.

**Comment by timbussmann** [comment]
> but the fact that after a probe a found license could not be converted or is expired should not DEBUG.

well, this might be specific to this situation. By default, NServiceBus probes several locations for licenses and ignores invalid/outdated licenses for situations where older licenses might be located in some folders. I think changing this logging behavior would generate a high amount of confusing noise in those cases.
I could see some potential in adjusting the error/warning message to indicate that there is additional information available when enabling debug log levels.

**Comment by ramonsmits** [comment]
I don't think it generated lots of noise as its *only* on startup and second only is ONE log event. On how many probe locations could there be data? 1 maybe 2 locations? If the data on these locations is valid then the behavior should be as is. 

1. Probe all locations for data (outcome no data, data, error). (DEBUG/TRACE)
2. No data or access error at no location, then create trial license and break
3. Load data until valid license found.
4. If no valid license found (authorization, invalid, expired, validation, or deserialization issue) show ONE ERROR showing all probed locations WITH DATA and their reasons for failure. (ERROR)
